docs/tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.md

```diff
@@ -0,0 +1,35 @@
+# Test duplicated registry entries
+
+This test will check that two monitored registries with the same name value but with different capitalization only triggers one added event when the registry is created.
+
+## General info
+
+|Tier | Number of tests | Time spent|
+|:--:|:--:|:--:|
+| 0 | 1 | 15s |
+
+## Test logic
+
+The test monitor two registries with the same path but different capitalization. It creates
+one registry with the path being one of the two registries monitorized and then tries to grab
+one added event for the registry creation. Finally it tries to grab the added event one second
+time but it should rise one TimeoutError to ensure only one added event was sent.
```

Comment:
```suggestion
# Test duplicated registry entries

This test will check that two monitored registries with the same name value but with different capitalization only triggers one added event when the registry is created.

## General info

|Tier | Number of tests | Time spent|
|:--:|:--:|:--:|
| 0 | 1 | 15s |

## Test logic

The test monitor two registries with the same path but different capitalization. It creates
one registry with the path being one of the two registries monitored and then tries to grab
one added event for the registry creation. Finally, it tries to grab the added event one second
time but it should raise one TimeoutError to ensure only one added event was sent.
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1247/commits/c89eb367365e53d034a3f04058598377b3acc2c6)